XPlat/AzureMessaging/iOS/samples/NotificationHubsSample/NotificationHubsSample.csproj

```diff
@@ -101,10 +105,7 @@
     <Compile Include="HomeViewController.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\source\Xamarin.Azure.NotificationHubs.iOS.csproj">
-      <Project>{99ECFF92-CA4E-4473-ADFC-5D7F3ABBC4C6}</Project>
-      <Name>Xamarin.Azure.NotificationHubs.iOS</Name>
-    </ProjectReference>
+    <PackageReference Include="Xamarin.Azure.NotificationHubs.iOS" Version="2.0.4" />
```

Comment:
Same as with the Android, we shouldn't use both project references and package references. I would suggest project references as the binding project is pretty simple and does not use a targets file. Then we don't need the nocache and co.

Reply:
Don't have both in here, and yeah we could just keep project reference, but I kind of like the fact that the sample is a real end to end test of the full package and not just a reference to the project.